src/main/window-manager.ts

```diff
@@ -181,8 +199,8 @@ export class WindowManager extends Singleton {
   }
 
   hide() {
-    if (this.mainWindow && !this.mainWindow.isDestroyed()) this.mainWindow.hide();
-    if (this.splashWindow && !this.splashWindow.isDestroyed()) this.splashWindow.hide();
+    if (!this.mainWindow?.isDestroyed()) this.mainWindow.hide();
+    if (!this.splashWindow?.isDestroyed()) this.splashWindow.hide();
```

Comment:
I think this should be left as it was or:
```suggestion
    if (!this.mainWindow?.isDestroyed()) this.mainWindow?.hide();
    if (!this.splashWindow?.isDestroyed()) this.splashWindow?.hide();
```
<img width="488" alt="Screen Shot 2021-04-26 at 12 31 43 PM" src="https://user-images.githubusercontent.com/40840436/116118544-6d56c480-a68b-11eb-99c5-a4dd0cfc3200.png">

Reply:
Ah yes, you are right